packages/shared/src/types/firestore.ts

```diff
@@ -315,6 +320,7 @@ export interface EmailAttachment {
 export interface EmailPayload {
   to: string;
   subject: string;
+  bcc: string;
```

Comment:
We probably want to add `emailFrom` here as well, since we've moved all of the email message fields to the cloud function (and have them stored as `payload` of email process document).

Also, update the JSDoc comment.